REFERENCE.md

```diff
@@ -694,6 +694,8 @@ A compatible vehicle is a vehicle that:
 2. belongs to the makes and models Smartcar supports, and
 3. supports the permissions.
 
+**\*Note:** the options (dict) argument is only valid for Smartcar API v1.0.
```

Comment:
I don't think this sentence applies here, the options dictionary allows for things like setting client id and client secret both of which are valid for v2.0